### DISCUSSION TITLE
Release 4.11.0

### DISCUSSION THREAD

**Discussion by sradner13** [go-acme/lego#1885] [Q&A] [answered]
Some new DNS providers, such as Google Domains, reference release 4.11.0.  Will this be available in the coming months? Patiently awaiting this version to makes it way into Traefik.

Cheers, and thanks for all the great work on these projects.

**Comment by ldez** [comment, marked as answer]
Hello,

I think we will create a release when the 2 PRs will be reviewed and merged
https://github.com/go-acme/lego/milestones/v4.11

**Comment by sradner13** [comment]
Thank you for the prompt response.  I'll watch the milestone discussion for
future updates.  À votre santé

**Comment by vorillaz** [comment]
Hi there @ldez, first things first congrats of your amazing work with this library, we are in the middle of integrating `go-acme` and one of the providers in the next release (Bunny) is in our watch list since we want to use it along with Traefik. 
Is there an ETA of the new release or maybe a minor version (4.10.1) in your plans before refactoring the two pending PRs?

Or maybe adjust the documentation as some listed providers are not released yet.

Once again thanks a ton :)

**Reply by ldez** [reply]
Hello, thank you.

a minor version is v4.11.0 (v4.X.0)
a patch version is v4.10.1 (v4.10.X)

We follow semver, based on the already merged PRs the next release will be v4.11.0 and it will also contain the PRs related to the [milestone v4.11](https://github.com/go-acme/lego/milestones/v4.11).

> Or maybe adjust the documentation as some listed providers are not released yet.

We cannot do that: the documentation is published when a PR is merged because if there is either a bug fix or an improvement of the doc, it must be updated.
In all the provider documentation pages you have a version (ex `Since: v4.11.0`).

---

You don't need a release to integrate lego as lib, as the branch `master` is stable then you can use it.
Obviously, I recommend using a release but if you have time constraints it's a solution.

**Comment by vorillaz** [comment]
Hi there @ldez, 
I am aware of the semver release and it you be amazing if you could do release a version with the new DNS providers.
Unfortunately we are indirectly using Lego as part of Traefik thus we cannot use the master branch version directly

**Reply by ldez** [reply]
https://github.com/traefik/traefik/pull/9883
